### PR TITLE
Web add more config options

### PR DESCRIPTION
**Pull request by shanehughes1990**

### CODE REVIEW

**Review by mklebrasseur — APPROVED** [review]

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -224,6 +229,15 @@ config.peopleSearchQueryTypes = ['user','conferenceRooms'];
 // Prejoin page.
 config.prejoinPageEnabled = {{ $ENABLE_PREJOIN_PAGE }};
 
+// Welcome page.
+config.enableWelcomePage = {{ .Env.ENABLE_WELCOME_PAGE }};
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:233]: Use the defined variables so $ENABLE_WELCOME_PAGE

**Review by saghul — CHANGES_REQUESTED** [review]

File: web/rootfs/defaults/settings-config.js (hunk 1 above)
Comment (shanehughes1990) [reply web/rootfs/defaults/settings-config.js:233]: Oh my bad, I will change that.

Comment (shanehughes1990) [reply web/rootfs/defaults/settings-config.js]: Done, I changed the 3 I added.

File: docker-compose.yml [hunk 2]
```diff
@@ -80,6 +83,7 @@ services:
             - RESOLUTION_WIDTH_MIN
             - START_AUDIO_ONLY
             - START_AUDIO_MUTED
+            - DISABLE_AUDIO_LEVELS
```
Comment (saghul) [inline docker-compose.yml:86]: Please sort it alphabetically.

File: web/rootfs/defaults/settings-config.js [hunk 3]
```diff
@@ -224,6 +229,15 @@ config.peopleSearchQueryTypes = ['user','conferenceRooms'];
...
 config.prejoinPageEnabled = {{ $ENABLE_PREJOIN_PAGE }};
 
+// Welcome page.
+config.enableWelcomePage = {{ $ENABLE_WELCOME_PAGE }};
+
+// Close page.
+config.enableClosePage = {{ $ENABLE_CLOSE_PAGE }};
+
+// Noisy mic detection notification.
+config.enableNoisyMicDetection = {{ $ENABLE_NOISY_MIC_DETECTION }};
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:239]: This probably belongs in the audio section. We don't have a rule per se, but I tried to group the options semantically, as a best effort.

Comment (shanehughes1990) [reply web/rootfs/defaults/settings-config.js:239]: Is there anything else? I can do it all at once

**Comment by shanehughes1990** [comment]
Was there anything else that needed done or is this good to merge now?

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Sorry for the delay, cheers!

**Comment by shanehughes1990** [comment]
I don't suppose we could maybe make a stable-5142-4 release? I kind of really wanna drop that config file with official release 👍

**Comment by saghul** [comment]
Sorry, it'll have to wait a bit. Busy with other things here...

**Pull request merged**